src/5.0_to_5.1/migrateSavedFilter.ts

```diff
@@ -0,0 +1,12 @@
+import { FilterState as FilterState51 } from "@activeviam/dashboard-base-5.1";
+import { MigrateFilterCallback } from "../migration.types";
+import { MdxString } from "@activeviam/mdx-5.0";
+
+/**
+ * Returns the 5.1 migrated version of the 5.0 `filterState`.
+ * Used only for saved filters, not for filters within dashboards.
+ */
+export const migrateSavedFilter: MigrateFilterCallback<
+  MdxString,
+  FilterState51<"serialized">
+> = (filterState) => ({ mdx: filterState });
```

Comment:
There is no need to migrate saved filters: if you look at what is saved in the content server it already has the wanted format.
![image](https://user-images.githubusercontent.com/29061952/210231988-d9f00200-b490-4aff-931a-2470006b94df.png)
So let's revert this commit.

Reply:
While the implementation needs to change (there is nothing to do), we might as well keep a dummy `migrateSaveFilter` just for the sake of setting a precedent, and benefitting from the changes to `parse` and `stringify` for compound identifiers that occurred between 5.0 and 5.1.